CHANGES.rst

```diff
@@ -2,11 +2,15 @@
  CHANGES
 =========
 
-4.0.1 (unreleased)
+4.1.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Support newer zope.configuration and persistent. See `issue 2
+  <https://github.com/zopefoundation/zope.preference/issues/2>`_.
```

Comment:
Does it still run with older versions or should the minimum versions in `setup.py` changed to these newer versions?

Reply:
It should still run with older versions.